jenkins-integration/beaker/install/pe/10_install_pe.rb

```diff
@@ -0,0 +1,3 @@
+test_name 'Install PE'
+
+install_pe
```

Comment:
I presume this beaker method already has support for picking a specific version, configured via env vars or something?